static/app/views/performance/queues/charts/latencyChart.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import {OrganizationFixture} from 'sentry-fixture/organization';
+
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import useOrganization from 'sentry/utils/useOrganization';
+import {LatencyChart} from 'sentry/views/performance/queues/charts/latencyChart';
+
+jest.mock('sentry/utils/useOrganization');
+
+describe('latencyChart', () => {
+  const organization = OrganizationFixture();
+  jest.mocked(useOrganization).mockReturnValue(organization);
+
+  let eventsStatsMock;
+
+  beforeEach(() => {
+    eventsStatsMock = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/events-stats/`,
+      method: 'GET',
+      body: {
+        data: [],
+      },
+    });
+  });
+  it('renders', () => {
+    render(<LatencyChart />);
+    screen.getByText('Avg Latency');
+    expect(eventsStatsMock).toHaveBeenCalledWith(
+      '/organizations/org-slug/events-stats/',
+      expect.objectContaining({
+        query: expect.objectContaining({
+          yAxis: [
+            'avg_if(span.self_time,span.op,queue.submit.celery)',
+            'avg_if(span.self_time,span.op,queue.task.celery)',
+            'count_op(queue.submit.celery)',
+            'count_op(queue.task.celery)',
+          ],
+        }),
+      })
+    );
+  });
```

Comment:
I _think_ it's a good idea here, if possible, to `await` when the loading spinner disappears, so the test can unmount cleanly, but worth asking the FE channel if that's important or not

Reply:
updated to await for loading spinners!